gef-ui/server/routes/supporting-information-upload.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express');
+const multer = require('multer');
+const { multerFilter } = require('../utils/multer-filter.utils');
+const {
+  uploadSupportingDocument, deleteSupportingDocument,
+} = require('../controllers/supporting-information/supporting-documents');
+const { validateRole, validateToken, validateBank } = require('../middleware');
+
+// The following routes cannot use the same csrf checks as the rest of the routes
+// as the Ministry of Justice multi-file-upload component does not allow passing tokens to the request body.
+// So we instead make a separate uploadCsrf token and check it here.
+const router = express.Router();
+
+// eslint-disable-next-line consistent-return
```

Comment:
Please remove inline Eslint rule.

Reply:
Please add function documentation.

Reply:
can you do return next() on line 27 to fix this?

Reply:
Yes! I'll do that for both of them, thanks.